script.js

```diff
@@ -0,0 +1,318 @@
+                        out+='<div class ="k-key '+ArrKey[i][j].code+'">'+ArrKey[i][j].en+'</div>';
+                    }else{
+                        out+='<div class ="k-key '+ArrKey[i][j].code+'">'+ArrKey[i][j].EN+'</div>';
+                    }
+                }
+                el.innerHTML = out;
+            });
+        }
+        if(key == 'en'){
+            let rows = document.querySelectorAll('.keyboard__row');
+            rows.forEach((el,i) => {
+                let out = '';
+                for(let j = 0; j < ArrKey[i].length; j++){
+                    out+='<div class ="k-key '+ArrKey[i][j].code+'">'+ArrKey[i][j].EN+'</div>';
+                }
+                el.innerHTML = out;
+            });
+        }
+    }else{
+        if(key == 'ru'){
+            let rows = document.querySelectorAll('.keyboard__row');
+            rows.forEach((el,i) => {
+                let out = '';
+                for(let j = 0; j < ArrKey[i].length; j++){
+                        out+='<div class ="k-key '+ArrKey[i][j].code+'">'+ArrKey[i][j].ru+'</div>';
+                }
+                el.innerHTML = out;
+            });
+        }
+        if(key == 'RU'){
+            let rows = document.querySelectorAll('.keyboard__row');
+            rows.forEach((el,i) => {
+                let out = '';
+                for(let j = 0; j < ArrKey[i].length; j++){
+                    if(ArrKey[i][j].letterRu == 1){
+                        out+='<div class ="k-key '+ArrKey[i][j].code+'">'+ArrKey[i][j].RU+'</div>';
+                    }else{
+                        out+='<div class ="k-key '+ArrKey[i][j].code+'">'+ArrKey[i][j].ru+'</div>';
+                    }
+                    
+                }
+                el.innerHTML = out;
+            });
+        }
+        if(key == 'EN'){
+            let rows = document.querySelectorAll('.keyboard__row');
+            rows.forEach((el,i) => {
+                let out = '';
+                for(let j = 0; j < ArrKey[i].length; j++){
+                    if(ArrKey[i][j].letterEn == 1){
+                        out+='<div class ="k-key '+ArrKey[i][j].code+'">'+ArrKey[i][j].EN+'</div>';
+                    }else{
+                        out+='<div class ="k-key '+ArrKey[i][j].code+'">'+ArrKey[i][j].en+'</div>';
+                    }
+                }
+                el.innerHTML = out;
+            });
+        }
+        if(key == 'en'){
+            let rows = document.querySelectorAll('.keyboard__row');
+            rows.forEach((el,i) => {
+                let out = '';
+                for(let j = 0; j < ArrKey[i].length; j++){
+                    out+='<div class ="k-key '+ArrKey[i][j].code+'">'+ArrKey[i][j].en+'</div>';
+                }
+                el.innerHTML = out;
+            });
+        }
+    }
+}
+
+
+PrintContent();
+
+PrintKeyboard(languageFlag, 0);
+
+
+
+//переключение языков и размера шрифта
+document.addEventListener('keydown',function(evt){

```

Comment:
please split up logic for language switching and capslock

there's no need re-render all keyboard when capslock is active, you can just printing capitalize letters

Reply:
anyway, it's not a good idea use one generic func for so different functionality